20FinalProject/index.html

```diff
@@ -1,10 +1,187 @@
+    <div class="page">
+        <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+            <div class="container">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#example-navbar-collapse">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="#hero1">Diana</a>
+                </div>
+
+                <!-- Collect the nav links for toggling -->
+                <div class="collapse navbar-collapse" id="example-navbar-collapse">
+                    <ul class="nav navbar-nav">
+                        <li>
+                            <a href="#hero1">Home</a>
+                        </li>
+                        <li>
+                            <a href="#me">About Me</a>
+                        </li>
+                        <li>
+                            <a href="#portfolio">Portfolio</a>
+                        </li>
+                        <li>
+                            <a href="#contactme">Contact Me</a>
+                        </li>
+                    </ul>
+                </div>
+                <!-- /.navbar-collapse -->
+            </div>
+            <!-- /.container -->
+        </nav>
+        <header id="hero1">
+            <div class="hello animated wobble">
+                <h1>HELLO</h1>
+            </div>
+            <h2>i am diana diaz</h2>
+            <p> Here is a collection of my work. Enjoy! </p>
+        </header>
+        <section id="me">
+            <div class="aboutme">
+                <article class="about">
+                    <h3>About Me</h3>
+                    <p>Hi, I am currently attending ACA San Antonio Campus. On my free time you can find me working on my coding skills while sipping on some coffee.</p>
+                </article>
+                <aside class="aboutimg"> <img src="img/1.png" alt="" /></aside>
+            </div>
+            <div class="mywork">
+                <aside class="workimg"> <img src="img/p-2.png" alt=""></aside>
+                <article class="work">
+                    <h3>My Work</h3>
+                    <p> Below you can browse through some of my work. My skills include responsive design, bootstrap and many more to come! </p>
+                </article>
+            </div>
+        </section>
+        <div id="portfolio">
+        <h2 class="portfolio">Portfolio</h2>
+      </div>
+        <section class="owl-carousel">
+            <div class="item animated">
+                <a href="https://dianadiaz.github.io/intro-workbook/05LessonFive/Checkpoint1/"><img src="img/p-1.png" alt="Berries Blog" /> </a>
```

Comment:
You want the user to stay on your site, right? (Yes, you do 😛 ). Add `target="_blank"` to your `a` tags